src/Cardano/Prelude/Base.hs

```diff
@@ -13,3 +21,23 @@ import           Numeric.Natural as X
 -- | Rename `id` to `identity` to allow `id` as a variable name
 identity :: Category cat => cat a a
 identity = id
+
+-- | Explicit output with @Text@ since that is what we want most of the time.
+-- We don't want to look at the type errors or warnings arising.
+putTextLn :: Text -> IO ()
+putTextLn = putStrLn
+
+-- Length which includes @Text@ as well as @Foldable@.
```

Comment:
There's a good solution for this called `mono-traversable`, why not bring that in? Then we can use `length` on ByteString as well.

Reply:
@parsonsmatt That's the approach (similar to) what Serokell used in `universum`, which is nice but seems kind of complicated to me - http://hackage.haskell.org/package/universum-1.5.0/docs/src/Universum.Container.Class.html#length
Can't we simply add an instance for `ByteString`? And for everything else we need?
Then again, I do trust your judgment so if you think we can add that, can you create a PR and see what the reaction is from other people? I don't mind if others agree.